src/Exception/RequestException.php

```diff
@@ -124,32 +124,7 @@ public static function create(
      */
     public static function getResponseBodySummary(ResponseInterface $response)
     {
-        $body = $response->getBody();
-
-        if (!$body->isSeekable() || !$body->isReadable()) {
```

Comment:
Wow... it seems that after I have added the function `get_message_body_summary` which mostly I copied from here... There was this added afterwards https://github.com/guzzle/guzzle/pull/2081 

@sagikazarmark or @Tobion can you release a 1.5.1 containing this change?